fastrace/examples/asynchronous.rs

```diff
@@ -36,6 +36,8 @@ async fn other_job() {
     }
 }
 
+// FIXME: Remove `expect` after the issue is fixed. https://github.com/rust-lang/rust-clippy/issues/13458
```

Comment:
Please allow the lint in `fastrace/Cargo.toml`

Reply:
done.